fontforgeexe/collabclientui.c

```diff
@@ -471,30 +471,32 @@ gpointer getSessionCallbacksObject()
 void collabclient_notifySessionJoining( cloneclient_t *cc, FontViewBase* fv )
 {
 #ifdef BUILD_COLLAB
-    
-    printf("AAA collabclient_notifySessionJoining() fv:%p\n", fv);
+
+    printf("AAA collabclient_notifySessionJoining() cc: %p, fv:%p\n", cc, fv);
```

Comment:
Ah, right. I've added in a warning message now (it pops up in that `Warnings` box)